src/coreComponents/physicsSolvers/fluidFlow/wells/WellControls.cpp

```diff
@@ -108,7 +108,7 @@ WellControls::WellControls( string const & name, Group * const parent )
     setDefaultValue( -1 ).
     setSizedFromParent( 0 ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "Global component densities of the injection stream [moles/m^3 or kg/m^3]" );
+    setDescription( "Defines the global component of the injected fluid in the reservoir as a fraction of its components [moles/m^3 or kg/m^3]" );
```

Comment:
I actually think these are mole fractions. @tjb-ltk ?
```suggestion
    setDescription( "Defines the global component fractions of the injected fluid." );
```

Reply:
Yes mole fractions...  @dkachuma where does the use of global come in...   I've seen this in geos a fair bit, and global implies a quantity over a lot of elements   ,  in this case I'm guessing it implies all components,  but seems a bit wordy to me..

Reply:
"Global" meaning overall including all phases. As opposed to say gas (or water) mole fractions.